test/unit/dom/appendChild.js

```diff
@@ -221,4 +221,31 @@ describe('dom: appendChild', () => {
   runTests('slot');
   runTests('host');
   runTests('root');
+
+  describe('fragment:', () => {
```

Comment:
Would it be possible (and worth the effort?) instead of manually creating this test just include DocumentFragment inside all the other tests? It seems like it's a matter of changing `switch/case` function and adding additional `runTests('documentFragment');`.